moon-openshift.yaml

```diff
@@ -617,123 +617,153 @@ objects:
               },
               "77.0": {
                 "image": "selenoid/vnc_chrome:77.0",
-                "port": "4444"
+                "port": "4444",
+                "privileged": true
```

Comment:
We will not add `--privileged` to default deployment manifests as too many users don't have such permission because of security concerns. Instead we are trying to either use CAP_SYS_ADMIN kernel capability or `--no-sandbox` flag.

Reply:
Delete privileged